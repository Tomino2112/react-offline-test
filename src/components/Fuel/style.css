.fuel h4:after {
    content: "%";
    font-size: 0.5em;
    vertical-align: top;
}