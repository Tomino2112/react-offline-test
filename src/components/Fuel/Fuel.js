import React from "react";
import "../../styles/icons.css";
import "./style.css";
import { ucFirst } from "../../utils";

const Fuel = ({ name, percentage }) => {
    return <div className="fuel">   
        <div className={`icon icon-${name}`}></div>
        <h4>{percentage}</h4>
        <h3>{ucFirst(name)}</h3>
    </div>
};

export default Fuel;