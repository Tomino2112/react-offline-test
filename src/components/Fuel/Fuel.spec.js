import React from "react";
import { shallow } from "enzyme";
import Fuel from "./Fuel";

describe("Fuel", () => {
    test("should render and match snapshot", () => {
        const wrapper = shallow(<Fuel name={"Foo"} percentage={"10.0"} />);

        expect(wrapper).toMatchSnapshot();
    });
});