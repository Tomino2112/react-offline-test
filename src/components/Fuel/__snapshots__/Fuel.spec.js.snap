// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fuel should render and match snapshot 1`] = `
<div
  className="fuel"
>
  <div
    className="icon icon-Foo"
  />
  <h4>
    10.0
  </h4>
  <h3>
    Foo
  </h3>
</div>
`;
